packages/linux-amd64/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "@cyclonedx/cdxgen-plugins-bin-linux-amd64",
+    "version": "1.0.0",
```

Comment:
Wrong version number. OS and architecture is also missing in the package.json file.